README.md

```diff
@@ -1,97 +1,58 @@
 
-Cut the connector off one one end of the RJ11/12 cable, then strip and connect the wires to the RS485 device's screw terminals. Pin 3 to B, Pin 4 to A, and Pin 5 to GND.  A and B are for data communication, and G is ground. 12V is left empty.
+## Required ESPHome Hardware
 
-*NOTE: Wire colors can vary from cable to cable. Ensure you are matching the pins as shown in the diagram below regardless of cable color, i.e. Pin 3 to B, Pin 4 to A, Pin 5 to G.*
+ESPHome-econet is a great first ESPHome project due to the easy hardware setup. All that's needed to run ESPHome-econet is an ESP32 or ESP8266 microcontroller and an RS485 Interface, plus a phone cord to hook it up to your Rheem appliance and a USB-C charger to power it. For simplicity, we recommend the m5Stack K045 Kit, which includes both the ESP32 & RS485 components in a simple package.
 
-```text
-    6P4C RJ11/RJ12 male connector end view   
-    
-               +---------+
-            1  ---       |
-            2  ---       +--+      
-        B   3  ---          |     
-        A   4  ---          |        
-       GND  5  ---       +--+
-            6  ---       |
-               +---------+
-```
+For full details on what hardware to buy and how to set it up, head over to the [Recommonded Hardware Purchase and Setup  page on our wiki](https://github.com/esphome-econet/esphome-econet/wiki/Recommended-Hardware-Purchase-and-Setup-Instructions).
 
-This picture shows a correctly wired setup. While wire colors can vary, this images shows the most common color layout reported by users (Green -> B, Red -> A, Black -> GND):
-<img src="https://github.com/esphome-econet/econet-docs/blob/main/photos/Correctly-Wired-K045.jpeg?raw=true" alt="A correctly wired K045 unit." width=50%>
+<img src="https://github.com/esphome-econet/econet-docs/blob/main/photos-and-screenshots/Correctly-Wired-K045.jpeg?raw=true" alt="A correctly wired K045 unit." width=25%>
 
 ## Software Installation
 
-Setup can be done from the command line using the esphome docker image, or via a (mostly) graphical interface by using an esphome dashboard installation; see Getting Started at <https://esphome.io/index.html> for full ESPHome installation and usage instructions. Initial installation must be done over USB. After this, updates and configuration changes can be made entirely over WiFi.
-
-Installation is a three-step process:
-
-1. Customize the basic configuration yaml for your device and environment
-2. Deploy to the device via command line or ESPHome Dashboard
-3. Add your device to Home Assistant
-
-### Step 1: Customizing the esphome-econet yaml for Your Device
+For simplified software installation, simply connect your ESP32 device to your computer via USB and [visit the Installation page on the ESPHome-econet website](https://esphome-econet.github.io/install/). Simplify select your Rheem hardware type and the site will allow you to image the device, connect it to wi-fi, and add it to Home Assistant.
```

Comment:
I had left you some comments at discord when you initially asked for feedback. You have a typo here: s/Simplify select/Simply select/ I think I mentioned another typo there too. Please check.